app/services/participants/base.rb

```diff
@@ -26,7 +26,7 @@ def call
 
   private
 
-    implement_instance_method :participant_profile_state, :perform_action!, :matches_lead_provider?
+    implement_instance_method :participant_profile_state, :participant_profile_states, :perform_action!, :matches_lead_provider?
```

Comment:
This is a service for switching participant state, right? Why do we need this change here?

Reply:
It was actually in the base class then I moved it to the user_profile as I think it's more appropriate to have it there.

Reply:
I mean `app/services/participants/base.rb` and `app/services/record_declarations/base.rb` are not the same, and I would only expect this PR to be touching the latter.